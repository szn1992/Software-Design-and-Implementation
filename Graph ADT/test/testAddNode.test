# Tests that multiple nodes can be added

# Create a graph
CreateGraph graph1

# Add multiple nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5
AddNode graph1 n6
AddNode graph1 n7
AddNode graph1 n8
AddNode graph1 n9
AddNode graph1 n10

# Print nodes in graph1
ListNodes graph1