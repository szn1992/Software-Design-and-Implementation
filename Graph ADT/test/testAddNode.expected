# Tests that multiple nodes can be added

# Create a graph
created graph graph1

# Add multiple nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1
added node n6 to graph1
added node n7 to graph1
added node n8 to graph1
added node n9 to graph1
added node n10 to graph1

# Print nodes in graph1
graph1 contains: n1 n10 n2 n3 n4 n5 n6 n7 n8 n9